awsemo/middlewares.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'm not sure about this one but we can experiment and try to remove it. It doesn't provide any functionality we need.